pandas/conftest.py

```diff
@@ -404,6 +404,10 @@ def indices(request):
     return indices_dict[request.param].copy()
 
 
+# Needed to generate cartesian product of indices
```

Comment:
this is a non-obvious name, call this index2_fixture

Reply:
Went with `index_fixture2` to be consistent with [nulls_fixture2](https://github.com/pandas-dev/pandas/blob/master/pandas/conftest.py#L267) and [unique_nulls_fixture2](https://github.com/pandas-dev/pandas/blob/master/pandas/conftest.py#L279)